src/routes/Viewport/AutoForwarder/index.tsx

```diff
@@ -22,6 +36,18 @@ const AutoForwarder = ({ docked }: { docked: Boolean }) => {
   const [running, setRunning] = useState(false);
   const { forwardUrls } = useForwardUrls(endpointId);
   const [url, setUrl] = useState('');
+  const { forwardWebhook } = useForwarder(endpointId);
```

Comment:
When endpointId is empty (i.e. ''), `useForwardUrls(endpointId)` is causing a GraphQL error, as it tries to fetch forward URLs for an endpoint that's empty.  Here is the request that's being sent:

![image](https://user-images.githubusercontent.com/504505/110995383-25741c00-832f-11eb-8f46-15b425e2284f.png)

This results in this response from our GraphQL server:

![image](https://user-images.githubusercontent.com/504505/110995616-826fd200-832f-11eb-879e-0aefa309cb44.png)

Also, every time I change the endpoint from the dropdown, the same request is being sent with an empty endpointId. You can test this by putting a breakpoint here, as that code will be called on every GraphQL error: https://github.com/webhook-buddy/webhook-buddy-client/blob/master/src/index.tsx#L72

You can also see this error being printed to the console:
![image](https://user-images.githubusercontent.com/504505/110995990-19d52500-8330-11eb-8a12-ffc9569f32ee.png)

To fix this, I suggest you extract this into a separate component, and only load it when an appropriate endpoint is selected:
```
        <label>Auto-forward to</label>
        <Autosuggest
          type="url"
          placeholder="Forward to URL (e.g. http://localhost:8000/send-webhook-here)"
          userInput={url}
          setUserInput={setUrl}
          suggestions={forwardUrls}
          position={AutosuggestPositionEnum.Up}
          disabled={running}
        />
      </div>
```

You may also want to extract the endpoint select list to a separate component as well to make it consistent.